x/migration/module/autocli.go

```diff
@@ -57,6 +57,7 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 					Long:           "Claim the account balance of the given Morse account address, by signing the message with the private key of the Morse account.",
 					PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "morse_src_address"}, {ProtoField: "morse_signature"}},
 					Skip:           true, // skipped because autoCLI cannot handle loading & signing using the Morse key.
+					// TODO_MAINNET_MIGRATION(@bryanchriswhite, #1034): Add morse account claiming CLI, incl. examples (see x/supplier/module/autocli.go).
```

Comment:
There is already a TODO for this in x/migration/module/cmd/claim_morse_account.go.
```suggestion
```